src/middlewared/middlewared/plugins/datastore/event.py

```diff
@@ -11,20 +12,9 @@ class Config:
 
     events = defaultdict(list)
 
-    @accepts(Dict(
-        "options",
-        Str("description", required=True),
-        Str("datastore", required=True),
-        Str("plugin", required=True),
-        Str("prefix", default=""),
-        Dict("extra", additional_attrs=True),
-        Str("id", default="id"),
-        Str("process_event", null=True, default=None),
-        strict=True,
-    ))
+    @api_method(DatastoreRegisterEventArgs, DatastoreRegisterEventResult, private=True)
```

Comment:
The service is already defined as private, `private=True` is (or should be) redundant